src/ZombieDriver.php

```diff
@@ -94,13 +94,14 @@ public function stop()
     public function reset()
     {
         $js = <<<JS
-browser.destroy();
-browser = null;
```

Comment:
Ah, then you've confused me in first place by saying that this popup case isn't handled. Then it is handled indirectly, since nobody have complained about it.

Since no problem can be spotted I'm closing this PR.